### ISSUE TITLE
Compatibility with MERSCOPE outputs

### ISSUE THREAD

**Issue by davidhbrann** [dcjones/proseg#5]
Hi,

Looks like a cool package and preprint. I tried to run `proseg detected_transcripts.csv --merfish` but encountered a couple of errors.

Here's an example of the `detected_transcripts.csv` file, which is outputed from the MERSCOPE software (and which in my hand also matches the format of the data in the vizgen `vz-ffpe-showcase` buckets from the data [link](https://info.vizgen.com/merscope-ffpe-solution) in your preprint. For more details, see page 45 in their user guide [here](https://vizgen.com/wp-content/uploads/2023/06/91600001_MERSCOPE-Instrument-User-Guide_Rev-G.pdf). 
```
,barcode_id,global_x,global_y,global_z,x,y,fov,gene,transcript_id,cell_id
242,118,9029.88,13003.255,0.0,270.0,1872.5022,0,Sult1d1,ENSMUST00000113314,-1
545,297,7444.5713,7944.677,6.0,1744.6135,1940.0,483,Epcam,ENSMUST00000234623,1997283000004200110
114,311,7393.4614,7934.633,6.0,1271.3741,1847.0,483,Blank-12,Blank-12,1997283000004200106
```

As you can see, a number of the columns don't match your `merfish` defaults (though maybe here merfish ≠ MERSCOPE). What I did was 1) remap the `-1` for unassigned cells to `0` since the command line wouldn't let me put `-1` as a value for --cell-id-unassigned and 2) convert the `transcript_id` name into an integer (with e.g. `pd.Categorical` or sklearn's `LabelEncoder`) column named `transcript`. [EDIT: I think the `barcode_id` column should also work]. Then I ran it as follows:

```
proseg -x 'global_x' -y 'global_y' -z 'global_z' \
--fov-column 'fov' --gene-column 'gene' --transcript-id-column 'transcript' \
 --cell-id-unassigned "0" --cell-id-column 'cell_id' \
detected_transcripts.csv
```

Does this workaround make sense? Are there any other settings I should change?  Would it be possible to add a preset for MERSCOPE? Or maybe add some more details to the README about running it on MERSCOPE data?

**Comment by dcjones** [comment]
This is very much my mistake for mis-documenting how I actually ran merscope data. In short, I was working with ffpe showcase data which was prior to them including a `cell_id` column. Proseg needs to be initialized with some prior segmentation, so I had to construct my own csv file with initial cell assignments, hence the preset being out of step with `detected_transcripts.csv`.

I'll put out a new version shortly to conform to vizgen's detected_transcripts format. In the mean time, what you're doing seems fine to me. Sorry about the extra hassle.
